src/App.tsx

```diff
@@ -14,7 +93,15 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie
+          handleTtitleSubmit={handleTtitleSubmit}
+          title={title}
+          handleTitleChange={handleTitleChange}
+          isTitleCorrect={isTitleCorrect}
+          movie={movie}
+          isMovieLoading={isMovieLoading}
+          handleAddMovie={handleAddMovie}
+        />
```

Comment:
Do you really need to pass so many props? Can't you create and use them in FindMovie component instead of App?